de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/RoleManagementTest.java

```diff
@@ -86,7 +94,10 @@ public void testAssignDisciplineToOtherByRepository() {
 		openRoleManagementEditor();
 		
 		// Add a disciple for the Repository
-		SWTBotTableItem newDisciplineTableItem = createNewDiscipline("Repository", null);
+		
+		SWTBotTable userTable = bot.table();
+		final String  username = userTable.cell(0, 1).replace("[", "").replace("]", "");

```

Comment:
Are we acutually using this variable? If not we should remove it... I think we don't need it anymore because we're just selecting the first user in the list down the way...